scripts/copper/bulk/prepare.py

```diff
@@ -16,19 +16,32 @@
 
 if __name__ == "__main__":
     bulk_copper = bulk("Cu", "fcc", 3.8)  #
-    data_path = Path("data/copper/bulk")
+    data_path_k = Path("data/copper/bulk_k")
+    data_path_ce = Path("data/copper/bulk_ce")
+    prepare_clean_directory(data_path_k)
 
-    prepare_clean_directory(data_path)
+    prepare_clean_directory(data_path_ce)
 
     calculators = list[Castep]()
 
+    # k points convergence test
     for n_k_points in range(1, 11):
-        config = CastepConfig(data_path / f"bulk_{n_k_points}", "bulk")
+        config = CastepConfig(data_path_k / f"bulk_{n_k_points}", "bulk")
         params = BulkOptimizationParams(n_k_points=n_k_points)
         calculator = get_bulk_optimization_calculator(bulk_copper, params, config)
         prepare_calculator_with_submit_script(calculator)
 
         calculators.append(calculator)
+    # cut off energy convergence test
```

Comment:
maybe extract each of these into a function, either in ppc library or just in this file